src/cli.jl

```diff
@@ -155,28 +196,28 @@ function write_rlm(filename, rlm)
 	end
 end
 
-function make_output_tree_names(nwk1, nwk2)
-	fn = [basename(nwk) for nwk in (nwk1, nwk2)]
-	name1, name2 = if fn[1] == fn[2]
-		name, ext = splitext(fn[1])
-		d1 = split(dirname(nwk1), '/')[end]
-		d2 = split(dirname(nwk2), '/')[end]
-		name1, name2 = if d1 == d2
-			name * "_1.resolved" * ext, name * "_2.resolved" * ext
-		else
-			name * "_$(d1).resolved" * ext, name * "_$(d2).resolved" * ext
-		end
-		@warn "The two input trees have the same filename. Writing output as:
-		$nwk1 --> $name1
-		$nwk2 --> $name2"
-		name1, name2
+function make_output_tree_names(nwk_names, ext; liberal=false)
```

Comment:
Is it a good idea to have output names depend on resolve type? This is a bit inconsistent with how the rest of TreeKnit behaves, *i.e.* output names are the same whatever the options were set to.